Task3/js/components/Zombie.js

```diff
@@ -0,0 +1,59 @@
+function Zombie(zombie) {
+    this.zombie = zombie || null;
+    this.zombie.health = this.zombie.health || 0;
+    this.zombie.healthCurrent = this.zombie.health;
+    this.zombie.healthPercent = 100;
+    this.zombie.death = false;
+    this.zombie.events = {
+        'create': function() {}
```

Comment:
'create' event name is logically incorrect
any zombie has no information what will happen if it dies
that's application logic

so, it'd be correct to name it 'dead' or 'die'
meaning "current zombie is dead"